bok_choy/browser.py

```diff
@@ -207,7 +207,7 @@ def browser_check_func():
             return False, None
 
     browser_instance = Promise(
-        browser_check_func, "Browser is instantiated successfully.", try_limit=3).fulfill()
+        browser_check_func, "Browser is instantiated successfully.", try_limit=3, timeout=60).fulfill()
```

Comment:
@jzoldak Should we up this to 100 seconds to ensure it's tried thrice? That's pretty long, but we're at the hands of selenium's 30s timeout in at least one case.